pscript/commonast.py

```diff
@@ -1034,6 +1034,12 @@ def _convert_Assign(self, n):
     def _convert_AugAssign(self, n):
         op = n.op.__class__.__name__
         return AugAssign(self._convert(n.target), op, self._convert(n.value))
+
+    def _convert_AnnAssign(self, n):
+        if n.value is None:
+            raise RuntimeError("Cannot convert AnnAssign nodes with no assignment!")
```

Comment:
I changed this to RuntimeError because that's what all other errors in this file were.